metpy/calc/basic.py

```diff
@@ -307,6 +307,13 @@ def coriolis_parameter(latitude):
         The corresponding coriolis force at each point
 
     """
+    try:
+        latitude = latitude.to('radians').m
+    except AttributeError:
```

Comment:
Would it be better to refactor this into a helper to use here and in `get_wind_components`?

Reply:
Like this:
```python
def _check_radians(value):
    try:
        value = value.to('radians').m
    except AttributeError:
        pass
if np.greater(np.nanmax(np.abs(value)), 2 * np.pi):
   warnings.warn(...)

return value
```